setup.py

```diff
@@ -312,8 +312,7 @@ def run(self):
             return
 
         if self.pkgconfigdir is None:
-            python_lib = sysconfig.get_python_lib(True, True,
-                                                  self.install_data)
+            python_lib = sysconfig.get_path("purelib")
```

Comment:
This doesn't return the same directory and also is ignoring the prefix.

Reply:
Distutils is deprecated as of Python 3.10 and is scheduled to be removed out of Python in 3.12. Let's make this a separate PR to get rid of distutils as a unit, I'll drop this commit here.